### ISSUE TITLE
Client Allocation Request for: Starboard Ventures

### ISSUE THREAD

**Issue by shaun97** [filecoin-project/filecoin-plus-client-onboarding#1430]
---

# Client Allocation Questions 

## Core Information
- Name: Starboard Ventures
- Website / Social Media: https://www.starboard.ventures/
- Region: Asia excl. Greater China
- DataCap Requested: 50TiB
- Addresses to be Notarized: f1gncpmfcwmgc7bvsmqauivrzv3tp7ety53k7nlaa
- Notary Requested: steven004

@shaun97 Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by steven004** [comment]
Hey @shaun97, thanks for your request. Can you kindly add more information / context here on:

1. yourself/ your organization (with links to official website/social media/news mentions etc)
2. Your history working on IPFS/Filecoin (notable contributions to the community, type of engagement shown over time etc)
3. Use case: type/source/quantity/diversity/scarcity of the data you plan to store on Filecoin
4. Miners you intend to allocate DataCap with allocation proportion(with redundancy/location/specific requirement of storage miners), if you are working with a service provider to store your data to Filecoin network, please name the provider.
5. How could people retrieve the data you store in Filecoin Network, do you have a tool/app for it?
6. What's your next plan for the data you plan to store in Filecoin Network? For-profit or not?
7. Would you be OK in posting deal transactions here?

Thanks

**Comment by shaun97** [comment]
Hello @steven004, thanks for the prompt reply. I've posted the answers to the questions below:

### _1. Yourself/ your organization (with links to official website/social media/news mentions etc)_
Starboard Ventures main business is to develop tools and applications to empower the blockchain data economy through 3rd-party (non-project and non-miner) data analysis, mechanism design and product development. We aim to improve the efficiency of data services in order to allow providers to provide data services more effectively, allowing network participants to have access to high-quality data, boosting the value of the web3 data economy network.

Currently, we provide professional consulting services and product solutions to help our clients build and develop successful economic networks.

Our expertise will bring values in below categories:

Design and experiment new economic networks;
Model and simulate existing designs to validate economic robustness and soundness;
Deploy and optimize softwares for infrastructures and applications in Web3 and the metaverse;
Perform network economic research and establish data analytic infrastructures for decision support in protocol governance and participation.

### **2. Your history working on IPFS/Filecoin (notable contributions to the community, type of engagement shown over time etc)**
Currently, we have a few applications providing information on the Filecoin network:

- ROI Calculator: https://sproi.starboard.ventures/
- Network Dashboard: https://dashboard.starboard.ventures/
- SPRD: https://sprd.starboard.ventures/

### **3. Use case: type/source/quantity/diversity/scarcity of the data you plan to store on Filecoin**
We plan to store publicly available large datasets and make it available for anyone.

### **4. Miners you intend to allocate DataCap with allocation proportion(with redundancy/location/specific requirement of storage miners), if you are working with a service provider to store your data to Filecoin network, please name the provider.**
We plan to allocate the data to around 6-10 different SPs based on our own evaluation of their reputation, with each miner storing between 10-15% of the data.

### **5. How could people retrieve the data you store in Filecoin Network, do you have a tool/app for it?**
We currently do not have an application for retrieval. We plan to leverage on the open dataset to experiment building data retrieval apps and other possible applications to build on the workflow.

### **6. What's your next plan for the data you plan to store in Filecoin Network? For-profit or not?**
We plan to let the public use these dataset if they require, providing a decentralised option for these publicly available datasets. This will be non-profit.

### **7. Would you be OK in posting deal transactions here?**
Yes

**Comment by steven004** [comment]
Hey @shaun97 , thanks for the contribution to the Filecoin ecosystem. I would like to support you on the real data landing in Filecoin network. 

I will grant you 5TiB for you to start your work. You could submit another application before it is used up. At the same time, please send me your business certificate, and the specific data type/content (what kind of public data and its source) information, then, we will go through a verification process before issuing you more datacap. 

Thanks.

**Comment by steven004** [comment]
Hey @shaun97 , Your Datacap Allocation Request has been approved

**Message sent to Filecoin Network**
> [bafy2bzaceakn73lawqaul4ax6vy72bzmaa6tad7lfhxpni66ragp7a645hi4m](https://filscan.io/tipset/message-detail?cid=bafy2bzaceakn73lawqaul4ax6vy72bzmaa6tad7lfhxpni66ragp7a645hi4m)

**Address**
> [f1gncpmfcwmgc7bvsmqauivrzv3tp7ety53k7nlaa](https://filscan.io/address/general?address=f1gncpmfcwmgc7bvsmqauivrzv3tp7ety53k7nlaa)

**Datacap Allocated**
> 5.0TiB

**Comment by shaun97** [comment]
Hello @steven004, 

Would it be possible to grant another datacap allocation of the same size to this address? I made a mistake of using a ledger to create my wallet instead of a cold wallet.

f1toz5izxdse43peqyd7zktmyqilvhf6u72z74gfq

What happened was that we intended to setup our own estuary node with a verified datacap to handle our deal-making. However, I realised afterwards that that it is not possible to use the Ledger with Estuary and hence had to recreate a cold wallet and request another datacap. 

Do let me know if you require anything else on my side! 

Thank you. 

Regards,
Shaun
